cli/src/cli.rs

```diff
@@ -400,23 +390,38 @@ pub async fn init_command(
     secret_manager: SecretManager,
     init_params: InitParameters,
 ) -> Result<Wallet, Error> {
-    let alias = if get_decision("Do you want to assign an alias to your wallet?")? {
-        Some(get_alias("New wallet alias").await?)
-    } else {
-        None
-    };
+    let mut address = init_params.address.map(|s| Bech32Address::from_str(&s)).transpose()?;
+    let mut forced = false;
+    if address.is_none() {
+        if get_decision("Do you want to set the address of the new wallet?")? {
+            address.replace(get_address("Set wallet address").await?);
+        } else {
+            forced = true;
+        }
+    }
+
+    let mut bip_path = init_params.bip_path;
+    if bip_path.is_none() {
+        if forced || get_decision("Do you want to set the bip path of the new wallet?")? {
```

Comment:
```
Do you want to set the address of the new wallet?: n
Set bip path (<coin_type>/<account_index>/<change_address>/<address_index>): n
cannot parse BIP path: n
```
That's a bit scam
Also I think we should at least display the default bip path for IOTA (4218/0/0/0) and Shimmer (4219/0/0/0) or add an option for the user to select the default or write a custom one (similar to how one can select to generate a mnemonic or provide a custom one)